ur_ros2_control_demos/urdf/ur5.urdf.xacro

```diff
@@ -0,0 +1,209 @@
+?xml version="1.0"?>
```

Comment:
Same comment about this urdf being a duplicate of the one in ur_description/urdf

Reply:
It is not the same, it has included ros2_control macro. Will hold it for know